superset-frontend/src/visualizations/FilterBox/controlPanel.jsx

```diff
@@ -20,6 +20,37 @@ import React from 'react';
 import { t } from '@superset-ui/core';
 import { sections } from '@superset-ui/chart-controls';
 
+const appContainer = document.getElementById('app');
+const bootstrapData = JSON.parse(appContainer.getAttribute('data-bootstrap'));
```

Comment:
I think that we usually parse bootstrapped data in the main index file of a given view. Do you think we could move those 2 lines higher in the files hierarchy as well?

Reply:
In `src/explore/index.jsx`

```
const exploreViewContainer = document.getElementById('app');
const bootstrapData = JSON.parse(
  exploreViewContainer.getAttribute('data-bootstrap'),
);
initFeatureFlags(bootstrapData.common.feature_flags);
```